src/schema/nmdc.yaml

```diff
@@ -147,18 +147,36 @@ classes:
       - CHMO:0000470
     slots:
       - eluent_introduction
+      - has_configuration
+    slot_usage:
+      id:
+        structured_pattern:
+          syntax: "{id_nmdc_prefix}:(dgms|omprc)-{id_shoulder}-{id_blade}{id_version}{id_locus}"
+          interpolated: true
+
+  Configuration:
+    abstract: true
+    class_uri: nmdc:Configuration
+    description: A platform that enables the user to configure the appearance, actions, and other usage preferences on a process.
```

Comment:
In what way is a configuration a "platform"?

Reply:
Generally we were thinking a computer platform.  We struggled with making this (and associated) descriptions and are open to other suggestions. Maybe "A framework that enables...."?

Reply:
I've updated this definition to "A programmable set of parameters that define the actions of a process and is shared among multiple instances of the process.".  I think that addresses some of the outstanding issues regarding understanding when we would use this class to hold configuration information.

Reply:
In the case of `MassSpectrometryConfiguration`, this would be 1:1 with a method file that we use to run the mass spectrometer.